sdcm/utils/version_utils.py

```diff
@@ -117,6 +118,12 @@ class ScyllaFileType(Enum):
 RepositoryDetails = namedtuple("RepositoryDetails", ["type", "urls"])
 
 
+@property
+def parse_scylla_version(version_to_parse: str) -> version.Version:
```

Comment:
we use also `SCYLLA_VERSION_RE` and `SCYLLA_VERSION_GROUPED_RE` from this file.
How about places that use it?

Reply:
they are being used in different places, to save the exact Scylla[-enterprise] version, including `build-id`, rather than this change, that will parse only the major and minor from the versions, for the sake of knowing if some features are supported in the current test setup or not